### PR TITLE
Update shellcheck test

### PR DESCRIPTION
**Pull request by jaapmarcus**

### CODE REVIEW

**Review by AlecRust — APPROVED** [review]
🚀 

**Edit:** Oh actually, looks like `Run ShellCheck script` is still succeeding with errors 🤔

**Comment by jaapmarcus** [comment]
> 🚀
> 
> **Edit:** Oh actually, looks like `Run ShellCheck script` is still succeeding with errors 🤔

It was supposed to fail wanted first the "error" warning to be working before I fix the code ...

**Comment by jaapmarcus** [comment]
Screenshot 2022-11-28 at 12 57 09

It is working now as intended... Will fix the issue now ..

**Review by AlecRust — APPROVED** [review]

**Comment by jaapmarcus** [comment]
Maybe need to check if we can use the docker image instead...

File: .github/workflows/lint.yml [hunk 1]
```diff
@@ -11,14 +11,14 @@ jobs:
...
-        run: sudo apt-get install -y shellcheck
+    - name: Checkout code
+      uses: actions/checkout@v3
 
-      - name: Run ShellCheck script
-        run: ./test/shellcheck.sh
+    - name: Run ShellCheck
+      uses: ludeeus/action-shellcheck@master
+      with:
+        version: v0.8.0
```
Comment (AlecRust) [inline .github/workflows/lint.yml]:
```suggestion
      uses: ludeeus/action-shellcheck@1.1.0
      with:
```

**Review by AlecRust — APPROVED** [review]

**Pull request merged**